crmsh/utils.py

```diff
@@ -2287,4 +2249,109 @@ def interface_choice():
     # should consider interface format like "ethx@xxx"
     interface_list = re.findall(r'(?:[0-9]+:) (.*?)(?=: |@.*?: )', out)
     return [nic for nic in interface_list if nic != "lo"]
+
+
+class InterfacesInfo(object):
+    def __init__(self, ipv6=False, second_heartbeat=False, specify_nic_list=[]):
```

Comment:
`specify_nic_list` name is a bit strange. We should find something better. Maybe `custom_nic_list` or simply `nic_list`

Reply:
Changed to `custom_nic_list`